pkg/apis/extensions/v1beta1/types.go

```diff
@@ -286,6 +286,12 @@ type DeploymentStatus struct {
 
 	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
+
+	// Total number of available pods (ready for at least minReadySeconds) targeted by this deployment.
+	AvailableReplicas int32 `json:"availableReplicas,omitempty"`
```

Comment:
@ironcladlou had a comment regarding having these as pointers to separate zero from unknown but I haven't seen any place where we use pointers in status so I am not sure about it. I think this is fine for now. We can come back later if it turns out to be a problem (I don't think it will).